specs/ppd-results.spec

```diff
@@ -0,0 +1,68 @@
+# Price Paid Data Search results
+
+Tags: search, PPD, Price paid data
+
+The Price paid data search results view
+
+## Deselect search queries
+
+Tags: results
+
+* Type "Rose Cottage" into the "Building name or number" field
+* Type "Plymouth" into the "Town or city" field
+* Clicking show results will route User to the search results page
+* Summary will display '21 transactions'
+* Clicking the cross icon next to 'Plymouth' will remove the search query parameter
+* Results page will re-render with new search parameters applied
+* Summary will display '9873 matching transactions'
+
+## Change settings
+
+Tags: results
+
+* Type "Rose Cottage" into the "Building name or number" field
+* Clicking show results will route User to the search results page
+* Clicking 'change search settings' will router User to search form
+* Prefilled value "Rose Cottage" visible in the "Building name or number" field
+
+## Download data
+
+Tags: results
+
+* Type "Rose Cottage" into the "Building name or number" field
+* Clicking show results will route User to the search results page
+* Clicking 'download data' will router User to the download page
+* Clicking 'get selected results as CSV' will prompt a file download of type CSV
+* Clicking 'get selected results as Turtle' will route User to generated TTL file
+* Clicking 'view SPARQL query' will route User to SPARQL query view with pre-populated search values
+* Clicking 'Back to results' will return you to the results page
+
+## Share
+
+Tags: results
+
+* Clicking 'share view' will open a modal dialogue prompting a Twitter link
+* The Users can close the modal dialogue by clicking the close cross
+
+## Results summary number displayed
+
+Tags: results
+
+* Summary will display 'Showing 100 transactions'
+* Clicking 'show a sample of at most 1000 results' will requery the search with new parameters
+* Results page will re-render with new search parameters applied
+* Summary will display 'Showing 1000 transactions'
+* Clicking 'show all results' will requery the search with new parameters
+* Results page will re-render with new search parameters applied
+* Summary will display 'Current selection: show all results'
+
+## Results list item
+
+Tags: results
+
+* A result item has a postal address as title
+* A result item has a list of transaction History with at least one or more item
+* Clicking the transaction history external link will route User to the Elda API populated with property transaction data
+* A result item has a list of address values
+* Clicking the address value magnifiying glass icon will requery the search with new parameters
```

Comment:
I don't know how easy it is to capture this, but the magnifying glass will specifically add the parameter that it's next to. So if, for example, the icon is on the "Town: Plymouth" row, clicking the icon will do a new search with the existing search terms plus "Town = Plymouth" as a constraint. It should also be the case that there will necessarily be fewer results (or the same number) compared to the current search, since we're adding a conjunctive constraint.